qiita_pet/handlers/processing_handlers.py

```diff
@@ -0,0 +1,27 @@
+from tornado.web import authenticated
+
+from .base_handlers import BaseHandler
+from qiita_ware.dispatchable import processor
+from qiita_db.parameters import ProcessedSortmernaParams
+from qiita_ware.context import submit
+
+
+class ProcessHandler(BaseHandler):
+    @authenticated
+    def post(self):
+        study_id = int(self.get_argument('study_id'))
+        preprocessed_data_id = int(self.get_argument('preprocessed_data_id'))
+
+        # We currently do not support defining the parameters
```

Comment:
Yes this is ready for another review. ETA 1-2 days.

2015-01-01 21:27 GMT+01:00 Antonio Gonzalez <notifications@github.com>:

> In qiita_pet/handlers/processing_handlers.py
> <https://github.com/biocore/qiita/pull/732#discussion-diff-22402078>:
>
> > @@ -0,0 +1,27 @@
> > +from tornado.web import authenticated
> > +
> > +from .base_handlers import BaseHandler
> > +from qiita_ware.dispatchable import processor
> > +from qiita_db.parameters import ProcessedSortmernaParams
> > +from qiita_ware.context import submit
> > +
> > +
> > +class ProcessHandler(BaseHandler):
> > +    @authenticated
> > +    def post(self):
> > +        study_id = int(self.get_argument('study_id'))
> > +        preprocessed_data_id = int(self.get_argument('preprocessed_data_id'))
> > +
> > +        # We currently do not support defining the parameters
>
> OK, do you have an ETA for that? Is this ready for another review?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/732/files#r22402078>.
>



-- 
Jose Navas